### PR TITLE
Adam-smithHTML/CSS_Portfolio

### PR DESCRIPTION
**Pull request by evansnyamekye**
This is a portfolio page of Adam smith. This website was developed with HTML and CSS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
     <script src="challenge.js"></script>
-    <title>Challenge</title>
+    <title>Adam Smith: Porfolio</title>
```
Comment (evansnyamekye) [inline index.html:10]: Kindly make changes or give the appropriate name for the title header before proceeding to the main HTML mockup. It is important to always give simple header titles to the title tag to make your code nice!

Thank you

File: styles.css [hunk 2]
```diff
@@ -30,5 +30,5 @@ h2{
   color: aqua;
 }
 .social-media-links{
-  color: #ffd540;
+  color: #fac816;
```
Comment (evansnyamekye) [inline styles.css:33]: Kindly change the text color #fac816 to something more visible that will enable the user to read well. The best practice to outline text in HTML mockups is to make the text color white or any color that will come out well with the background. Doing this makes your website more fun to read on any info displaced. 

Many thanks

File: index.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
     <script src="challenge.js"></script>
-    <title>Challenge</title>
+    <title>Adam Smith: Porfolio</title>
   </head>
   <body>
     <h1>Adam Smith</h1>
```
Comment (evansnyamekye) [inline index.html:13]: Kindly use Prettier or the default text format contained in the HTML mockup to properly arrange your code. This enables your code to be clean. 

thanks

**Review by evansnyamekye — COMMENTED** [review]
#Changes required ♻️ 

Kindly check my comments below and make all the required changes. 
Submit for your review when you are done. 

Many Thanks 😀

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @evansnyamekye,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Very professional and complete comments. ✔️
 - Very polite comments. ✔️ 
 - Good use of inline comments. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — COMMENTED** [review]
Hi @evansnyamekye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite and provide a reason :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — COMMENTED** [review]
Hi @evansnyamekye 👍, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Used GitHub inline comment and summary features ✔️ 
Give reasons for suggested changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**